samples/depthai_sample/face_detection/face_detection.py

```diff
@@ -35,10 +37,8 @@
 parser.add_argument(
     "-nn",
     "--nn_model",
-    help="select model path for inference",
-    default=os.path.join(
-        os.path.dirname(__file__), "models/face_detection_yunet_120x160.blob"
-    ),
+    help="Provide model name or model path for inference",
+    default="face_detection_yunet_160x120",
     type=str,
```

Comment:
```suggestion
    type=Path,
```

FYI: 直接 Path インスタンスを作れます。